File: src/models/Wallet.js
export class Wallet {
  constructor({ walletId, userId, blockchain, address, balance = 0, lastUpdated }) {
    this.walletId = walletId;
    this.userId = userId;
    this.blockchain = blockchain;
    this.address = address;
    this.balance = balance;
    this.lastUpdated = lastUpdated || new Date();
  }
}

File: src/models/wallet.py
from pydantic import BaseModel
from datetime import datetime
from typing import Optional

class WalletCreate(BaseModel):
    blockchain: str
    address: str

class Wallet(BaseModel):
    wallet_id: str
    user_id: str
    blockchain: str
    address: str
    balance: float = 0
    last_updated: datetime = datetime.now()